### PR TITLE
ipsec: move styles to theme

### LINKED ISSUE #2695: IPSec widget not following css.
User points out that in the IPSec widget there are hard codec colours, in the <script> section.

### PR DESCRIPTION
**Pull request by fabianfrz**
move all styles to sass file and recompile the CSS

closes #2695

### CODE REVIEW

**Comment by fabianfrz** [comment]
@AdSchellevis our them developers usually fix those issues in approximately a week to a maximum of two releases. For that time the patch can be hold back on master until the themes are ready but that is up to you.

**Comment by AdSchellevis** [comment]
@fabianfrz there are way more theme's around then the ones publicly available, hence the major release switch. Other parties should have the opportunity to change their products in time.

**Comment by fabianfrz** [comment]
Then I would suggest keeping it on master until the next major release, the public templates can update (it would not have any impact because the selectors / properties) would not change anything because the element style is higher weighted then the class selector.

**Comment by marjohn56** [comment]
I've made no changes to Rebellion, but I now see this. On 18.7.2:
image

On 19.1.a_187 With @fabianfrz patch:

image

Don't think  I need to do anything.

File: src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -884,3 +884,20 @@ div[data-for*="help_for"] {
 .dashboard_grid_column table {
   table-layout: fixed;
 }
+
+$ipsec_bright_color: #EEEEEE;
+$ipsec_dark_color: #777777;
+
+#ipsec {
+  #ipsec-mobile, #ipsec-tunnel, #ipsec-Overview {
```
Comment (fichtner) [inline src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss]: I'd prefer this to be lowercase all the way

Comment (fabianfrz) [reply src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss]: me too but the ids aren't mine ;)

Comment (fichtner) [reply src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss]: but you touch the same lines so check mate! :P

**Comment by marjohn56** [comment]
I'm just heading out of the door so manyana. :)

**Comment by fichtner** [comment]
unfortunately this needs to be redone due to conflicts

**Comment by fabianfrz** [comment]
@fichtner do I have to do that or the theme team?

**Comment by fichtner** [comment]
good question. I'm not even sure what @AdSchellevis prefers as a solution here. the firewall live log has embedded classes now that can be safely overwritten...

**Comment by AdSchellevis** [comment]
we probably better opt for a similar solution as the firewall live log, embed css, let the theme overwrite if needed. 

@fabianfrz do you have time to create a pull request for that?

**Comment by fabianfrz** [comment]
@AdSchellevis done - see the new PR

**Pull request closed without merging**